sru/__init__.py

```diff
@@ -6,3 +6,4 @@
 """
 from sru.version import __version__  # noqa: F401
 from sru.modules import SRU, SRUCell  # noqa: F401
+from sru.modules import SRUpp, SRUppCell  # noqa: F401
```

Comment:
```
from sru.version import __version__ 
from sru.modules import SRU, SRUCell
from sru.modules import SRUpp, SRUppCell

__all__ = ['__version__', 'SRU', 'SRUCell', 'SRUpp', 'SRUppCell']
```
(this way you dont need the `# noqa` markings, I believe).

Reply:
will do.